packages/sitecore-jss/src/native-fetcher.ts

```diff
@@ -15,6 +15,10 @@ type NativeDataFetcherOptions = {
    * Optional request timeout.
    */
   timeout?: number;
+  /**
+   * optional headers to use in fetch calls
+   */
+  headers?: Record<string, string>;
```

Comment:
Shouldn't need this here, it's already on `RequestInit` type (which is part of final `NativeDataFetcherConfig` - see below).

Reply:
oh yeah, you're right